### ISSUE TITLE
TabsExtra won't close all tabs if a .png file in open in Sublime regession of #64

### ISSUE THREAD

**Issue by aireilly** [facelessuser/TabsExtra#67]
On Subl Build 4150, TabsExtra won't close all tabs if an image is open. Possible regression of #64?

**Comment by facelessuser** [comment]
Yeah, I worked on this recently and thought I had it fixed, it appears I did not. Sublime changed something under the hood and I've been fighting this. I'll take another look soon.

**Comment by facelessuser** [comment]
This is what we currently do. Linux seems to love to make this increasingly more difficult to abstract: https://github.com/facelessuser/MarkdownPreview/blob/master/browser.py#L60.

**Comment by aireilly** [comment]
NP! This is probably my most used plugin. This is a minor annoyance at best. Thanks for looking in to it.
